src/js/views/maps/SearchInputView.js

```diff
@@ -0,0 +1,167 @@
+"use strict";
+
+define([
+  "backbone",
+  "text!templates/maps/search-input.html",
+], (
+  Backbone,
+  Template,
+) => {
+  /**
+   * @class SearchInputView
+   * @classdesc SearchInputView is a shared component for searching information in the
+   * map toolbar.
+   * @classcategory Views/Maps
+   * @name SearchInputView
+   * @extends Backbone.View
+   * @since x.x.x
+   * @constructs SearchInputView
+   */
+  const SearchInputView = Backbone.View.extend({
+    /**
+     * The type of View this is
+     * @type {string}
+     */
+    type: "SearchInputView",
+
+    /**
+    * The HTML classes to use for this view's element
+    * @type {string}
+    */
+    className: "search-input",
+
+    /**
+     * The HTML classes to use for this view's HTML elements.
+     * @type {Object<string,string>}
+     */
+    classNames: {
```

Comment:
Curious why you made classNames an attribute of the view here, but a constant in other views?

Reply:
I def just copied the code from somewhere else haha. updated to constant since I fought for it🙂